setup_board

```diff
@@ -27,8 +27,10 @@ DEFINE_boolean getbinpkg "${FLAGS_TRUE}" \
   "Download binary packages from remote repository."
 DEFINE_string getbinpkgver "" \
   "Use binary packages from a specific version."
+
+get_board_and_variant $FLAGS_board $FLAGS_variant
```

Comment:
`FLAGS_board` and `FLAGS_variant` are not yet evaluated at this point, no? It happens below, after the line with `# Parse command line flags`, I think.

Reply:
True. Currently it only outputs the defaults. Since `get_board_and_variant` is only called to populate `BOARD` for the succeeding help text I'll just change the help text and remove `get_board_and_variant`.

Reply:
The BOARD variable is used below also for other things, so I don't think `get_board_and_variant` call should be completely removed. Just called after the flags are evaluated, like it used to be.